app/src/main/java/uk/ac/ebi/atlas/search/AutocompleteController.java

```diff
@@ -85,6 +86,15 @@ public String fetchMetaDataSuggestions( @RequestParam(value = "query") String qu
                 query,species.split(","))));
     }
 
+    @GetMapping(value = "/json/suggestions/gene_search",
```

Comment:
If I understand correctly, the functionality of both endpoints is the same at the moment and it's just the name of the endpoint which is different. So why can't we change the endpoint to the `gene_search` now in the front-end? and have only one endpoint in the backend?

Reply:
I'm not sure, @alfonsomunozpomer suggested this approach so that we can update the front end in a feasible time. I think if we replace it suddenly with the new one,  it might impact other components which are dependent only on the backend.